SFBaseKit/Extensions/UIKit/UIStackView+ShowAndHideArrangedSubview.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  UIStackView+ShowAndHideArrangedSubview.swift
+//  SFBaseKit
+//
+//  Created by Aleksandar Gyuzelov on 15.02.21.
+//  Copyright © 2021 Upnetix. All rights reserved.
+//
+
+import UIKit
+
+extension UIStackView {
```

Comment:
It is nice you have extracted this to the SFBaseKit. Could you also add documentation why those methods are helpful (mainly due to using stackViews in reusable cells).